### PR TITLE
String extensions sample notebook

### PR DESCRIPTION
**Pull request by ElisendaGascon**

### CODE REVIEW

File: Corvus.Extensions.Samples/StringExtensionsSample.dib [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+return url.EscapeContentType();
+
+#!markdown
+
+### `FromBase64()`
+
+#!markdown
+
+Decode a string from a base64-encoded byte array with the specified text encoding.
+
```
Comment (jamesbroome) [inline Corvus.Extensions.Samples/StringExtensionsSample.dib:115]: What's the default? The example doesn't specify a text encoding, so what is being used?

File: Corvus.Extensions.Samples/StringExtensionsSample.dib [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+Unescape a content type string.
+
+#!markdown
+
+#### Examples
+
+#!csharp
+
+string myString = "https%3A%60%60www%7Cyoutube%7Ccom%60watch%3Fv%3Dpcc-EXzK3vI";
+return myString.UnescapeContentType();
```
Comment (jamesbroome) [inline Corvus.Extensions.Samples/StringExtensionsSample.dib:166]: Same comment - this isn't a content type string

File: Corvus.Extensions.Samples/StringExtensionsSample.dib [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+#!markdown
+
+#### Examples
+
+The example below converts a string's byte representation in Unicode to a base 64 representation.
+
+#!csharp
+
+string hello = "Hello";
+return hello.AsBase64(Encoding.ASCII);	string encoded = hello.AsBase64(Encoding.Unicode);
```
[the remark this answers is not in the document]
Comment (jesshill99) [reply Corvus.Extensions.Samples/StringExtensionsSample.dib:39]: Fixed ✅

File: Corvus.Extensions.Samples/StringExtensionsSample.dib [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+
+#!markdown
+
+#### Examples
+
+#!csharp
+
+string url = "https://www.youtube.com/watch?v=pcc-EXzK3vI";
+string urlEncoded = url.Base64UrlEncode();
+    urlEncoded
```
[the remark this answers is not in the document]
Comment (jesshill99) [reply Corvus.Extensions.Samples/StringExtensionsSample.dib:58]: Fixed ✅

File: Corvus.Extensions.Samples/StringExtensionsSample.dib [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+
+### `EscapeContentType()`
+
+#!markdown
+
+Escape a content type string.
+
+#!csharp
+
+string url = "https://www.youtube.com/watch?v=pcc-EXzK3vI";
```
[the remark this answers is not in the document]
Comment (jesshill99) [reply Corvus.Extensions.Samples/StringExtensionsSample.dib:105]: Fixed ✅

File: Corvus.Extensions.Samples/StringExtensionsSample.dib (hunk 2 above)
Comment (jesshill99) [reply Corvus.Extensions.Samples/StringExtensionsSample.dib:166]: Fixed ✅

File: Corvus.Extensions.Samples/StringExtensionsSample.dib (hunk 1 above)
Comment (jesshill99) [reply Corvus.Extensions.Samples/StringExtensionsSample.dib:116]: Default is UTF8 but I've changed the example to specify an encoding (Encoding.ASCII)

**Review by jamesbroome — APPROVED** [review]

**Pull request merged**